lection3/task3.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+function concatenated(n) {
+    n = Number(n); // Для ввода через консоль ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯
```

Comment:
Could be added a checks that passed argument should be an positive integer value, min 1 and max 9